modules/inject-io/src/main/scala/io/janstenpickle/trace4cats/inject/io/IOLocalTraceInstances.scala

```diff
@@ -0,0 +1,16 @@
+package io.janstenpickle.trace4cats.inject.io
+
+import cats.effect.{IO, IOLocal}
+import io.janstenpickle.trace4cats.Span
+import io.janstenpickle.trace4cats.base.context.Local
+import io.janstenpickle.trace4cats.base.optics.Lens
+import io.janstenpickle.trace4cats.inject.Trace
+
+trait IOLocalTraceInstances {
+
+  /** Construct a `Trace[IO]` instance backed by the given `IOLocal[Ctx]`. */
+  def ioLocalTrace[Ctx](rootCtx: IOLocal[Ctx])(implicit lens: Lens[Ctx, Span[IO]]): Trace[IO] = {
```

Comment:
@ybasket I made `Lens.id` implicit, so no need for `ioLocalTrace(rootSpan: IOLocal[Span[IO]])`